src/helpers/fetches.spec.js

```diff
@@ -7,14 +7,15 @@ const client = new CozyClient({})
 
 describe('fetchContactsToUpdate', () => {
   it('should returns only contact to update', async () => {
-    const lastSuccess = new Date()
     const contactToUpdate = johnDoeContact
     const contactUpToDate = updateIndexFullNameAndDisplayName(johnDoeContact)
```

Comment:
Est-ce qu'on devrait pas tester le cas ou le contact a une date inférieure / supérieure à ce qu'on attend pour vérifier le comportement ?

Reply:
As we mock the query result and the date is used in the query definition, we can't test that